.github/workflows/algolia/src/index.ts

```diff
@@ -121,9 +84,22 @@ export const uploadAlgoliaData = async (records) => {
     process.env.AGOLIA_WRITE_KEY
   );
 
-  const index = client.initIndex("hash_learn");
+  const index = client.initIndex("hash_learn_testing1");
 
   return await index
     .saveObjects(records, { autoGenerateObjectIDIfNotExist: true })
```

Comment:
Looks like we don’t need `autoGenerateObjectIDIfNotExist` because `objectID: `${type}/${matterData.data.slug}`,`